3rdparty/CEGUI/CMakeLists.txt

```diff
@@ -71,6 +72,9 @@ if (MSVC) # only MSVC needs to download and build dependencies of CEGUI
     )
     message(STATUS "Installing done")
 
+else()
+    find_package(EXPAT REQUIRED)
```

Comment:
the final deliverable (when we actually merge) needs those build on Linux as well, or at the very least EXPAT (since freetype is ubiquitous).

If for some reason its not appropriate to build these ourselves, then they should at the very least be statically linked.

Reply:
I can just remove MSVC check and let all compilers build the deps. Since the deps source is already portable.